tests/behat/features/ra_export.feature

```diff
@@ -0,0 +1,28 @@
+Feature: A RAA manages tokens tokens registered in the selfservice portal
+  In order to manage tokens
+  As a RAA
+  I must be able to manage second factor tokens from my institution
```

Comment:
Please rewrite the feature outline to match the behavior that is actually tested in this feature.